ql/src/test/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/TestHiveMetaStoreAuthorizer.java

```diff
@@ -411,4 +411,5 @@ public void testU_DropDataConnector_authorizedUser() {
       fail("testU_DropDataConnector_authorizedUser() failed with " + e);
     }
   }
+
```

Comment:
Nit: Can you please remove this blank line?

Reply:
Done.